app/src/main/java/de/danoeh/antennapod/view/viewholder/EpisodeItemViewHolder.java

```diff
@@ -105,7 +105,7 @@ public void bind(FeedItem item) {
         isInQueue.setVisibility(item.isTagged(FeedItem.TAG_QUEUE) ? View.VISIBLE : View.GONE);
         container.setAlpha(item.isPlayed() ? 0.5f : 1.0f);
 
-        ItemActionButton actionButton = ItemActionButton.forItem(item, true, true);
+        ItemActionButton actionButton = ItemActionButton.forItem(item, true);
```

Comment:
As this is the only use of that method, could you please also remove the other parameter?

Reply:
Yea good idea!